jenkins-integration/lib/puppet/gatling/config.rb

```diff
@@ -65,3 +66,8 @@ def hiera_datadirs(hiera)
     [localpath, datadir]
   end
 end
+
+# Returns the path to the local r10k YAML configuration file
+def r10k_configpath(r10k)
+  File.join('config', 'r10ks', r10k + '.yaml')
```

Comment:
`r10ks` is a directory full of r10k configuration files, following the existing pattern of `scenarios/`, `nodes/`, and `hieras/`